src/app/login/login-form/login-form.component.html

```diff
@@ -0,0 +1,37 @@
+<form class="login-form">
+  <h3 [textContent]="submitValue === 'Sign In'
+    ? 'Login with existed account'
+    : 'Please create new account or use existed'"></h3>
+  <div class="form-group">
+    <label for="email" class="control-label text-capitalize">User email</label>
+    <input id="email"
+           placeholder="email"
+           type="text"
```

Comment:
maybe it would be better to set type="email" for autocompletion?

Reply:
yep, I missed it